README.md

```diff
@@ -293,6 +270,239 @@ See event-component.ts in apitest for a complete example of broadcasting user me
 - The EventHub is restricted to `userBroadcast` events, other event types will not be broadcasted between windows.
 - This means the you have to take care of propagating state changes between windows yourself.
 
+### Binding Components
+
+Golden Layout binds to components and then controls their position, size and visibility (positioning) so that they fit within a layout. There are 4 ways Golden Layout can bind to a component:
+
```

Comment:
Minor point, but maybe worth overthinking:
Maybe we should have a little table that is Embedded/Virtual on one axis and Registration/Events on the other axis and introduce these concepts separated from each other. 

|              | Embedded | Virtual |
|--------------|----------|---------|
| Registration | (a)      | (b)     |
| Events       | (c)      | (d)     |

where (a) to (d) are links to the next subsections

Reply:
Good idea but decided its superfluous and did not add. Can be inferred from bullets below. Also Readme is already quite dense so did not want add unnecessary.